borderify/manifest.json

```
@@ -1,14 +1,23 @@
 {
 
   "description": "Adds a solid red border to all webpages matching mozilla.org. See https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Examples#borderify",
-  "manifest_version": 2,
+  "manifest_version": 3,
   "name": "Borderify",
   "version": "1.0",
   "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/borderify",
   "icons": {
     "48": "icons/border-48.png"
   },
 
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "borderify@example.com",
```

Looks like the prevailing pattern in for the examples in this repo is `<example-name>@mozilla.org` where `example-name` is a kebab case identifier that resembles the example's parent directory. This pattern has the benefit of being [reserved on AMO](https://github.com/mozilla/addons-server/blob/72a130277cc3567cdfaf3785f2b2ca1f0c7a3353/src/olympia/constants/base.py#L379), so developers cannot submit add-ons with these IDs while testing.

```suggestion
      "id": "borderify@mozilla.org",
```